python/pip-wheel-metadata/pyarrow.dist-info/LICENSE.txt

```diff
@@ -0,0 +1,87 @@
+
```

Comment:
The whole `pip-wheel-metadata` directory looks like it shouldn't be in this PR, IMHO.